src/test/java/com/yahoo/validatar/AppTest.java

```diff
@@ -42,6 +42,8 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import static org.mockito.Mockito.*;
```

Comment:
Do we want star imports in tests? Also, maybe I missed it, but where do we use mockito in this test file?